git_machete/cmd.py

```diff
@@ -1339,9 +1340,9 @@ def get_combined_remote_sync_status(b: str) -> Tuple[int, Optional[str]]:
 # Reflog magic
 
 
-REFLOG = Tuple[str, str]
+REFLOG_ENTRY = Tuple[str, str]
 
-reflogs_cached: Optional[Dict[str, Optional[REFLOG]]] = None
+reflogs_cached: Optional[Dict[str, Optional[REFLOG_ENTRY]]] = None
```

Comment:
```suggestion
reflogs_cached: Optional[Dict[str, Optional[List[REFLOG_ENTRY]]]] = None
```

and possibly then you can remove `type: ignore`s below